### DISCUSSION TITLE
How about FastSam in Video

### DISCUSSION THREAD

**Discussion by DiamondGlassDrill** [kadirnar/segment-anything-video#68] [Ideas]
How about using FastSAM as Instance Segmenter Model? (50x faster than SAM)
https://github.com/CASIA-IVA-Lab/FastSAM

Thanks in advance.

**Comment by kadirnar** [comment]
I will review this. Thank you.

**Reply by DiamondGlassDrill** [reply]
How about an even more advanced and even faster way :)

With DINO and MobilSAM
https://github.com/IDEA-Research/Grounded-Segment-Anything/tree/main/EfficientSAM

Only MobilSAM (5x faster than FastSAM ;) )
https://github.com/ChaoningZhang/MobileSAM

**Reply by kadirnar** [reply]
Thank you for your interest. We will add these two models.

https://github.com/kadirnar/segment-anything-video/issues/82
